app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt

```diff
@@ -134,8 +137,12 @@ class KaigiAppScaffoldState @OptIn(ExperimentalMaterial3Api::class) constructor(
     val navController: NavHostController,
     val drawerState: DrawerState,
 ) {
-    fun navigate(drawerItem: DrawerItem) {
-        navController.navigate(drawerItem.navRoute)
+    fun navigate(drawerItem: DrawerItem, selectedDrawerItem: DrawerItem) {
+        navController.navigate(drawerItem.navRoute) {
+            popUpTo(selectedDrawerItem.navRoute) {
```

Comment:
I have a question 🙋 
If you go,
A -> B -> C -> D
And If you click B in the drawer, it will be
A -> B
Right?

Reply:
🤔 no, it should be only B

Reply:
I understood it now. Thanks.
Let's try anyway 👍  If I think the previous one is better, let me revert this at the time🙏